src/Workers/WorkerCommands.php

```diff
@@ -26,6 +29,8 @@ public function __construct(Application $console, Client $client, WorkerProvider
         $this->client = $client;
         $this->workerProvider = $workerProvider;
         $this->outputInterfaceAware = $outputInterfaceAware;
+        
+        $this->messageAdapterFactory = null;
```

Comment:
The `messageAdapterFactory` is mandatory, why using a setter instead of injecting it ?

Out of the box it is generating a fatal error

```
Catchable fatal error: Argument 1 passed to Puzzle\AMQP\Commands\Worker\Run::setMessageAdapterFactory() must be an instance of Puzzle\AMQP\Workers\MessageAdapterFactory, null given, called in /var/www/app/vendor/puzzle/amqp/src/Workers/WorkerCommands.php on line 46 and defined in /var/www/app/vendor/puzzle/amqp/src/Workers/MessageAdapterFactoryAware.php on line 10
```

The only thing that we can change in the `MessageAdapterFactory` is the `BodyFactory`.

The `WorkerCommands` could get the `BodyFactory` as a dependency, so it would be able to build the `MessageAdapterFactory`.